third-party/xen/Mybuild

```diff
@@ -0,0 +1,4 @@
+package third_party
+
+@Build(stage=1,script="$(EXTERNAL_MAKE) BUILD_DIR=$(abspath $(EXTERNAL_BUILD_DIR))/third_party/xen download install")
```

Comment:
Can you remove setting `BUILD_DIR`? 
I think you'll get `$(EXTERNAL_BUILD_DIR))/third_party/xen_headers` implicitly. Is something wrong with it?